### PR TITLE
Add documentation for AEMET OpenData integration

### PR DESCRIPTION
**Pull request by Noltari**
## Proposed change
Add documentation for AEMET OpenData integration: https://github.com/home-assistant/core/pull/45074

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45074
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2157
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/aemet.markdown [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: AEMET OpenData
+description: Instructions on how to integrate AEMET OpenData within Home Assistant.
+ha_release: 2021.2
```
Comment (MartinHjelmare) [inline source/_integrations/aemet.markdown]:
```suggestion
ha_release: 2021.3
```

File: source/_integrations/aemet.markdown [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+| API Key              | API Key from the website                                                                                                                                                                                                                   |
+| Name                 | Name of the integration                                                                                                                                                                                                                    |
+| Latitude             | Latitude for weather forecast and sensor                                                                                                                                                                                                   |
+| Longitude            | Longitude for weather forecast and sensor                                                                                                                                                                                                  |
+| Mode                 | Forecast mode, `daily` for daily forecast up to 6 days or `hourly` for an hourly forecast up to 3 days. |
```
Comment (MartinHjelmare) [inline source/_integrations/aemet.markdown]:
```suggestion
```

File: source/_integrations/aemet.markdown [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The integration creates weather entity and also sensors for all available conditions.
```
Comment (MartinHjelmare) [inline source/_integrations/aemet.markdown]:
```suggestion
The integration creates one weather entity per forecast mode and also sensors for all available conditions. There are two forecast modes, `daily` for daily forecast up to 6 days or `hourly` for an hourly forecast up to 3 days.
```

File: source/_integrations/aemet.markdown [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+For each condition `sensor` entity will be created with id:
```
Comment (MartinHjelmare) [inline source/_integrations/aemet.markdown]:
```suggestion
For each condition a `sensor` entity will be created with id: 
```

**Review by MartinHjelmare — DISMISSED** [review]
:+1:

File: source/_integrations/aemet.markdown [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+| API Key              | API Key from the website                                                                                                                                                                                                                   |
+| Name                 | Name of the integration                                                                                                                                                                                                                    |
+| Latitude             | Latitude for weather forecast and sensor                                                                                                                                                                                                   |
+| Longitude            | Longitude for weather forecast and sensor                                                                                                                                                                                                  |
```
Comment (frenck) [inline source/_integrations/aemet.markdown]: Use `%{ config_basic %}` for this instead.

File: source/_integrations/aemet.markdown [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  - Weather
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@noltari'
+ha_domain: aemet
+---
+
+The `aemet` weather platform uses [AEMET OpenData](https://opendata.aemet.es/) as a source for current meteorological data for your location. AEMET stands for `Agencia Estatal de Meteorología`, which is the Spanish Meteorological Agency.
```
Comment (frenck) [inline source/_integrations/aemet.markdown]:
```suggestion
The AEMET weather platform uses [AEMET OpenData](https://opendata.aemet.es/) as a source for current meteorological data for your location. AEMET stands for "Agencia Estatal de Meteorología", which is the Spanish Meteorological Agency.
```

File: source/_integrations/aemet.markdown [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+There is currently support for the following device types within Home Assistant:
+
+- Sensor
+- Weather
+
+You need an API key, which is free, but requires a [registration](https://opendata.aemet.es/centrodedescargas/altaUsuario).
+The AEMET OpenData service is completely free, but is limited to the Spanish territory.
```
Comment (frenck) [inline source/_integrations/aemet.markdown]:
```suggestion
You need an API key, which is free but requires a [registration](https://opendata.aemet.es/centrodedescargas/altaUsuario).
The AEMET OpenData service is completely free but is limited to the Spanish territory.
```

File: source/_integrations/aemet.markdown [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+| Latitude             | Latitude for weather forecast and sensor                                                                                                                                                                                                   |
+| Longitude            | Longitude for weather forecast and sensor                                                                                                                                                                                                  |
+
+The integration creates one weather entity per forecast mode and also sensors for all available conditions. There are two forecast modes, `daily` for daily forecast up to 6 days or `hourly` for an hourly forecast up to 3 days.
+
+For each condition a `sensor` entity will be created with id:
+
+`sensor.<integration name>_<monitored condition>`
+
+All conditions:
```
Comment (frenck) [inline source/_integrations/aemet.markdown]: The `entity_id` is not guaranteed, we should therefore not list it as such.

```suggestion
For each condition, the following sensors are created:
```

File: source/_integrations/aemet.markdown [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+| Condition             | Description                          |
+| :-------------------- | :----------------------------------- | 
+| `condition`           | Current weather condition code.      |
```
Comment (frenck) [inline source/_integrations/aemet.markdown]: Let's use normal names for the conditions.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @Noltari!

A couple of small comments and suggestions, could you please take a look? Thanks! 👍

File: source/_integrations/aemet.markdown [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+## Configuration
+
+To add AEMET OpenData integration go to **Configuration** >> **Integrations** and find the integration in the list.
+
+| Parameter | Value                                     |
+| :-------- | :---------------------------------------- |
+| api_key   | API Key from the website                  |
+| name      | Name of the integration                   |
+| latitude  | Latitude for weather forecast and sensor  |
+| longitude | Longitude for weather forecast and sensor |
```
Comment (frenck) [inline source/_integrations/aemet.markdown]:
```suggestion
{% configuration_basic %}
API Key:
  description: "API Key from the website"
Name:
  description: "Name of the integration"
Latitude:
  description: "Latitude for weather forecast and sensor"
Longitude:
  description: "Longitude for weather forecast and sensor".
{% configuration_basic %}
```

**Review by frenck — APPROVED** [review]
Thanks @Noltari!

**Pull request merged**